scripts/fish/functions/kc.fish

```diff
@@ -1,4 +1,8 @@
 function kc --argument-names context --description "Switch current kubernetes context"
-    set -l config (kubesess -v $context context) 
+    set -l cmd kubesess context
+    if test -n "$argv"
+        set -a cmd -v $context
+    end
+    set -l config (command $cmd) || return $status
```

Comment:
What variable is $status? A global variable for fish or did we miss to declare it?

Reply:
It’s a magic variable indicating the exit code of the previous command.